utils/Timestamp.py

```diff
@@ -0,0 +1,235 @@
+import audioowl
+import librosa
+import numpy as np
+from pydub import AudioSegment
+from pydub.utils import which
+import time
+from random import randint
+
+
+
+def find_best_sync_point(bottom_file_beats, top_file_beats, max_mix_sample, offset, mode):
```

Comment:
Maybe move this method to a sub script called timestamp_utils.py because this isn't what the app actually uses/sees. Assuming that timestamp() is the main method that other components will see and use